src/ApiCallJS.js

```diff
@@ -356,8 +361,16 @@ export async function verifyTwoFactor(code) {
     const data = { verification_code: code };
 
     await sendToEndPoint(config, 'PUT', url, accessToken, data).then(
-      function (responseText) {
-        resolve(responseText);
+      async function (responseText) {
+        await EncryptedStorage.setItem(appId + user_auth, JSON.stringify({
+          access_token: responseText.access_token,
+          client_id: responseText.client_id,
+          refresh_token: responseText.refresh_token,
+        }));
+        resolve({
```

Comment:
Mentioned in other PR, but let's revert this to `resolve(responseText)` and tidy app to use what it needs